packages/trader/src/Modules/Trading/Helpers/contract-type.tsx

```diff
@@ -13,6 +14,9 @@ type TItem = {
     value: string;
 };
 
+export const isMajorPairsSymbol = (checked_symbol: string, active_symbols: ActiveSymbols) =>
+    active_symbols.some(({ submarket, symbol }) => /major_pairs/i.test(submarket) && checked_symbol === symbol);
```

Comment:
would this also get translated?

Reply:
Yep, submarket is not translated to other languages, it's remain as 'major_pairs'